zeroconf.py

```diff
@@ -2018,7 +2033,7 @@ def __init__(
         self._GLOBAL_DONE = False
         self.unicast = unicast
 
-        self._listen_socket, self._respond_sockets = create_sockets(interfaces, unicast, ip_version)
+        self._listen_socket , self._respond_sockets, self._respond_socket_options = create_sockets(interfaces, unicast, ip_version)
```

Comment:
Per my comment above about discarding this value, we could also do something like this:
```python
self._listen_socket , self._respond_sockets, socket_options = create_sockets(interfaces, unicast, ip_version)
self_.respond_socket_options = self_respond_socket_options + socket_options
```
This will keep the `create_sockets` function more pure.